implementations/rust/ockam/ockam/src/protocols/mod.rs

```diff
@@ -23,6 +23,7 @@ impl ProtocolPayload {
     /// In order to easily decode incoming `ProtocolPayload`s, it is
     /// recommended to use the `ProtocolParser` abstraction, which handles
     /// matching between different decoders based on the protocol ID.
+    #[allow(dead_code)]
```

Comment:
I think protocol-related code shouldn't have landed in this PR

Reply:
This is from develop rebase